README.md

```diff
@@ -1,59 +1,85 @@
-next-big-thing $ ls
-
+~ $ mkdir my-git-project //Creates new directory
+~ $ cd my-git-project //Moves into the newly created directory
 ```
 
-You created a new directory with `mkdir next-big-thing` and then moved into it by changing your directory with `cd next-big-thing`. You typed `ls` to see all the files and folders in your brand new project, and, as expected, it was empty. But this directory, `next-big-thing`, is where you're going to put all your code.
-
-The first step is to transform this directory into a Git-enabled directory, which will allow it to keep track of all the changes to your code and interact with other Git remotes such as GitHub. How do we accomplish this? It's as simple as `git init`.
+## State How to Manage Files in a Directory With `git init`
 
-`git init` initializes a new Git repository in your current directory. Type `git init` from within your `next-big-thing` project directory.
+Now that we're in the directory where we want `git` to watch for changes (adding,
+removing, and editing files) let's set up this directory by _initializing_ it.
+In our terminal type:
 
 ```
-next-big-thing $ git init
-Initialized empty Git repository in /Users/avi/next-big-thing/.git/
+my-git-project $ git init
+Initialized empty Git repository in /Users/avi/my-git-project/.git/
 ```
 
-After entering `git init`, the output in your terminal reads `Initialized empty Git repository in /your/path/here/next-big-thing/.git/`. Git is letting you know that it created a new repository within the hidden `.git` folder in `next-big-thing`. This hidden directory, `.git`, is what Git uses to keep track of all the different versions of your code.
+After entering `git init`, the output in our terminal reads `Initialized empty Git repository in /our/path/here/my-git-project/.git/`. Git is letting we know that it created
+a new repository within the hidden `.git` folder in `my-git-project`. This hidden
+directory, `.git`, is what git uses to keep track of our log of changes. So, don't
+go in there and start randomly deleting things!
 
-You don't need to do anything with the `.git` directory –– it'll remain hidden. Just remember that if a directory has a `.git` directory inside of it, it is controlled by Git. It's also nice to know there's no magic behind the process: if Git is keeping track of multiple versions of files that you can't see, Git has to be putting them somewhere inside your computer. That somewhere is `.git`.
+## Use `git status` to Check the Status of Our Repository 
+
+Now we have git watching this directory, let's see what git can tell us about
+the directory with `git status`. Without any files added yet, we'll see:
+
+```
+On branch master
 
-At this point, though, there's nothing for Git to keep track of. We'll have to create some files and folders in our project first.
+No commits yet
 
-## Checking the status of your repository with `git status`
+nothing to commit (create/copy files and use "git add" to track)
+```
```

Comment:
Maybe "As expected, `git` confirms that it's not watching or _tracking_ anything in this directory. Let's change that!"